roadmap.md

```diff
@@ -2,7 +2,7 @@
 
 **Deliverable**|**Status**|**Expected Delivery**|**Description**
 :-----:|:-----:|:-----:|:-----:
-Msal Node GA|In Progress|February 2021|Our Msal Node library is currently in public preview. GA will include meeting our testing bar, fixing bugs from customers using public preview, and fleshing out our documentation. 
+Msal Node GA|Completed|February 2021|Our Msal Node library is currently in public preview. GA will include meeting our testing bar, fixing bugs from customers using public preview, and fleshing out our documentation. 
```

Comment:
Do we just want to remove completed items? If not and we want to leave it on for a while let's also update the description to reflect current state?

Reply:
I think we should keep past items.

Reply:
The rest of the description is fine for now, I'm liking going to rewrite the descriptions during my update pass.